package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbeat",
-  "version": "8.1.1",
+  "version": "8.1.21",
```

Comment:
Should this be `8.1.2`?

Reply:
Considering the tag made on backbeat, I went for `8.1.21` as the last `8.1` release is `8.1.20`.

Also weirdly the tag `8.1.2` does not exist, so your argument would be valid, but as we had `8.1.(2 + x)` and so on releases I suggest we go for `8.1.21`

Reply:
cool. I just thought maybe this is a typo. :)

Reply:
Knowing myself there was a 50% chance (at least) that this would be a typo.